internal/api/v1/appchart/fullindex.go

```diff
@@ -0,0 +1,29 @@
+package appchart
+
+import (
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/api/v1/response"
+	"github.com/epinio/epinio/internal/appchart"
+	apierror "github.com/epinio/epinio/pkg/api/core/v1/errors"
+
+	"github.com/gin-gonic/gin"
+)
+
+// FullIndex handles the API endpoint GET /appcharts
+// It lists all the known appcharts in all namespaces
+func (hc Controller) FullIndex(c *gin.Context) apierror.APIErrors {
```

Comment:
There is only one "index" and it's the full one. I will rename this to just "Index" (the file too)